packages/components/src/components/hds/code-editor/index.hbs

```diff
@@ -0,0 +1,53 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: MPL-2.0
+}}
+
+<div class={{this.classNames}} ...attributes>
+  {{#if (and this.isSetupComplete (or this.hasToolbarButton (has-block)))}}
+    <div class="hds-code-editor__header">
+      <div class="hds-code-editor__header-content">
+        {{yield
+          (hash
+            Title=(component "hds/code-editor/title")
+            Description=(component "hds/code-editor/description")
+            Button=(component "hds/code-editor/button")
+          )
+        }}
+      </div>
+
+      {{#if this.hasToolbarButton}}
+        <div class="hds-code-editor__header-toolbar">
```

Comment:
Can we change this name to "hasActions?"

This is how it looks like in the figma properties
<img width="258" alt="image" src="https://github.com/user-attachments/assets/139bee78-8ecf-44bf-923a-3dbd5c393c02" />

Reply:
(hasActions is the container -- which would potentially become `hds-code-editor__header-actions`)